src/screens/gameScreen.ts

```diff
@@ -0,0 +1,48 @@
+import { Entity } from '../entities/entity';
+import { ScreenElement } from './screenElement';
+
+/**
+ * Represents a game screen in the application.
+ * @interface
+ */
+export interface GameScreen {
+  /**
+   * Indicates whether the game screen is currently active.
+   * @type {boolean}
+   */
+  isActive: boolean;
+
+  /**
+   * Indicates whether the game screen is currently displayed.
+   * @type {boolean}
+   */
+  isDisplayed: boolean;
```

Comment:
What's the difference between isDisplayed and isActive? Seems like these usually have the same value.

Reply:
`isDisplayed` determines whether a screen will be rendered. `isActive` determines whether the screen is interactive and has it's logic running. This allows screens to be rendered over each other, ex. partially transparent pause screen(`isDisplayed=true` `isActive=true`) rendered over game screen(`isDisplayed=true` `isActive=false`).